scala/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.spark</groupId>
   <artifactId>spark-clr</artifactId>
-  <version>${spark.version}-SNAPSHOT</version>
+  <version>1.4.1-SNAPSHOT</version>
```

Comment:
Copied warning below. Since we will map each branch to a specific spark version at a given time - a constant spark-version in pom.xml is still managable.

[WARNING]
[WARNING] Some problems were encountered while building the effective model for com.microsoft.spark:spark-clr:jar:1.4.1-SNAPSHOT
[WARNING] 'version' contains an expression but should be a constant. @ com.microsoft.spark:spark-clr:${spark.version}-SNAPSHOT, c:\gitsrc\SparkCLR\scala\pom.xml, line 5, column 12